dom/src/main/scala/org/scalajs/dom/MediaQueryList.scala

```diff
@@ -8,23 +8,36 @@ package org.scalajs.dom
 
 import scala.scalajs.js
 
-/** A MediaQueryList object maintains a list of media queries on a document, and handles sending notifications to
-  * listeners when the media queries on the document change.
+/** A MediaQueryList object stores information on a media query applied to a document, with support for both immediate
+  * and event-driven matching against the state of the document.
   */
 @js.native
-trait MediaQueryList extends js.Object {
+trait MediaQueryList extends EventTarget {
 
-  /** true if the document currently matches the media query list; otherwise false. Read only. */
+  /** A boolean value that returns true if the document currently matches the media query list, or false if not. */
   def matches: Boolean = js.native
 
-  /** The serialized media query list */
+  /** A string representing a serialized media query. */
   var media: String = js.native
 
-  /** Adds a new listener to the media query list. If the specified listener is already in the list, this method has no
-    * effect.
+  /** Adds to the MediaQueryList a callback which is invoked whenever the media query status—whether or not the document
+    * matches the media queries in the list—changes.
+    *
+    * This method exists primarily for backward compatibility; if possible, you should instead use addEventListener() to
+    * watch for the change event.
+    * @deprecated
     */
+  @deprecated("Use addEventListener() instead", "2.3.0")
```

Comment:
Unfortunately 2.3.0 was released last night 😅 

```suggestion
  @deprecated("Use addEventListener() instead", "2.4.0")
```